src/seedsigner/main.py

```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/env python
+
+import argparse
+import logging
+import sys
+
+from seedsigner.controller import Controller
+
+logger = logging.getLogger(__name__)
+
+
+def main(sys_argv=None):
+    parser = argparse.ArgumentParser()
+    parser.add_argument(
+        "-l",
+        "--loglevel",
+        choices=list((logging._nameToLevel.keys())),
+        default="INFO",
+        type=str,
+        help='Set the log level (default: %(default)s)',
+    )
+
+    args = parser.parse_args(sys_argv)
+    logging.basicConfig(
```

Comment:
That there is no "filename" param, does this guarantee (even into the future) that messages logged will always end up on stdout?

Reply:
hm... how to guarantee this? The Python stdlib logging framework since forever logs to stderr if nothing else is given to the `logging.basicConfig()` call. A change of behaviour like logging to a not further specified file would be a huge break for many many applications that handle secrets, credentials, tokens etc.

Added a unit test in https://github.com/SeedSigner/seedsigner/pull/558/commits/a1d3489ad244f956527990f5556542b2a7c0b710 that actually verifies that the log is written to the stderr.

Reply:
stderr, of course, thank you for the correction.

And that it would raise alarms elsewhere eases my mind.

Reply:
yay. the added testcase is now green. and I also replaced the `logging.basicConfig` with explicit calls to have better control over what is exactly setup here.